src/NServiceBus.Persistence.DynamoDB.Tests/Serialization/MapperTests.cs

```diff
@@ -654,9 +660,234 @@ public void Should_not_throw_for_non_cyclic_references()
         [Test]
         public void Should_throw_for_non_object_root_types()
         {
-            var exception = Assert.Throws<InvalidOperationException>(() => Mapper.ToMap(new List<int>()));
+            var exception = Assert.Throws<SerializationException>(() => Mapper.ToMap(new List<int>()));
 
             Assert.That(exception.Message, Does.StartWith("Unable to serialize the given type"));
         }
+
+        [Test]
+        public void Should_respect_default_options_on_to_map()
```

Comment:
nitpick: this isn't really about respecting the default options but rather respecting the options argument, which just happens to be the default in this case.

Reply:
When I wrote default I meant the json serializer option default but yeah the term is ambiguous